tools/mbpp/trim_tasks.py

```diff
@@ -0,0 +1,48 @@
+import json
+import re
+
+from rich.console import Console
+
+from evalplus.data.mbpp import get_mbpp_plus, mbpp_serialize_inputs
+from evalplus.data.utils import write_jsonl
+
+
+def find_links(task_id: str, text: str) -> list:
+    url_pattern = re.compile(r"https?://\S+[.\S]+")
+    matches = url_pattern.findall(text)
+    return matches
+
+
+def main(args):
+    console = Console()
+    plus_problems = get_mbpp_plus(mini=False)
+
+    removed = []
+
+    for task_id, task in plus_problems.items():
+        # trim tasks with links in prompt
+        prompt = task["prompt"]
+        matches = find_links(task_id, prompt)
+        if matches:
+            console.print(f"Removed {task_id} for having links in prompt.")
+            removed.append(task_id)
```

Comment:
Instead of removing the whole task -- let's only simplify its prompt according to #156 -- e.g., remove the URL part in the prompt.

This change in prompt is fine as it is still aligned with the base MBPP given that this was brought by MBPP-sanitized.